xwiki-commons-core/xwiki-commons-displayer/src/main/java/org/xwiki/displayer/HTMLDisplayerManager.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.displayer;
+
+import java.lang.reflect.Type;
+import java.util.Map;
+
+import org.xwiki.component.annotation.Role;
+import org.xwiki.stability.Unstable;
+
+/**
+ * Provides a way to access the HTML Displayer for the given type.
```

Comment:
You can access the HTML displayer for a given type by injection or using the component manager. Why should I use this manager instead?

Reply:
So you don't have to reimplement the logic of `DefaultDisplayerManager#getHTMLDisplayer` each time you want to retrieve an HTML displayer:
https://github.com/xwiki/xwiki-commons/blob/c0868ea92abe1b1fbd14e6e1047e83bdefd61ae9/xwiki-commons-core/xwiki-commons-displayer/src/main/java/org/xwiki/displayer/internal/DefaultDisplayerManager.java#L62-L103

Reply:
I think what @mflorea meant is that it's not the only way to access the HTML Displayer if you already known which one you want. It's just a helper.

Reply:
Yes, my question was more for the Javadoc. I would mention that if you know precisely what type you want then you can inject directly the corresponding displayer. Otherwise, if you need the fall back on the generic displayer then you should use this component.